library/alloc/src/string.rs

```diff
@@ -2749,6 +2777,14 @@ impl FusedIterator for Drain<'_> {}
 #[cfg(not(no_global_oom_handling))]
 #[stable(feature = "from_char_for_string", since = "1.46.0")]
 impl From<char> for String {
+    /// Allocates an owned [`String`] from a single character.
+    ///
+    /// # Example
```

Comment:
Maybe I'm mistaken but isn't this usually always written as `Examples` even if it may only be one example?
```suggestion
    /// # Examples
    ///
```

Reply:
I ran a quick `grep`, and I found a lot of `# Example`. I don't think this is standardized.

<details><summary>Output</summary><p>

```
grep -r "# Example" | grep -v "# Examples"
backtrace/src/backtrace/mod.rs:/// # Example
backtrace/src/symbolize/mod.rs:/// # Example
backtrace/src/symbolize/mod.rs:/// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/ops/arith.rs:    /// # Example
core/src/num/dec2flt/mod.rs:/// # Example
core/src/num/f64.rs:    /// # Example
core/src/num/error.rs:/// # Example
core/src/num/error.rs:/// # Example
core/src/num/f32.rs:    /// # Example
core/src/ptr/metadata.rs:/// # Example
core/src/ptr/metadata.rs:/// # Example
core/src/ptr/mod.rs:/// # Example
core/src/hint.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/iter.rs:/// # Example
core/src/slice/mod.rs:    /// # Example
core/src/iter/adapters/peekable.rs:    /// # Example
core/src/iter/traits/iterator.rs:    /// # Example
core/src/task/poll.rs:    /// # Example
core/src/pin.rs://! # Example: self-referential struct
core/src/pin.rs://! # Example: intrusive doubly-linked list
core/src/pin.rs:    /// # Example
core/src/alloc/global.rs:/// # Example
core/src/array/mod.rs:    /// # Example
core/src/array/mod.rs:    /// # Example
core/src/option.rs:    /// # Example
core/src/mem/maybe_uninit.rs:    /// # Example
core/src/mem/maybe_uninit.rs:    /// # Example
core/src/mem/maybe_uninit.rs:    /// # Example
std/src/io/buffered/mod.rs:    /// # Example
std/src/io/buffered/mod.rs:    /// # Example
std/src/io/buffered/bufwriter.rs:/// # Example
std/src/io/buffered/bufreader.rs:    /// # Example
std/src/io/mod.rs:    /// # Example
std/src/io/mod.rs:    /// # Example
std/src/io/mod.rs:    /// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/collections/hash/map.rs:/// # Example
std/src/ffi/c_str.rs:    /// # Example
std/src/os/unix/net/ancillary.rs:/// # Example
std/src/os/unix/net/ancillary.rs:    /// # Example
std/src/os/unix/net/ancillary.rs:    /// # Example
std/src/os/unix/net/ancillary.rs:    /// # Example
std/src/os/unix/net/ancillary.rs:    /// # Example
std/src/os/unix/io.rs:    /// # Example
std/src/os/unix/io.rs:    /// # Example
std/src/os/unix/io.rs:    /// # Example
alloc/src/vec/drain.rs:/// # Example
alloc/src/vec/into_iter.rs:/// # Example
alloc/src/vec/drain_filter.rs:/// # Example
alloc/src/vec/splice.rs:/// # Example
alloc/src/string.rs:    /// # Example
alloc/src/string.rs:    /// # Example
alloc/src/string.rs:    /// # Example
alloc/src/sync.rs:    /// # Example
alloc/src/sync.rs:    /// # Example
alloc/src/sync.rs:    /// # Example
alloc/src/sync.rs:    /// # Example
alloc/src/sync.rs:    /// # Example
alloc/src/rc.rs:    /// # Example
alloc/src/rc.rs:    /// # Example
alloc/src/rc.rs:    /// # Example
alloc/src/rc.rs:    /// # Example
alloc/src/rc.rs:    /// # Example
alloc/src/rc.rs:    /// # Example
```

</p>
</details>